src/photosui.cs

```diff
@@ -480,4 +481,92 @@ interface PHProjectTypeDescriptionInvalidator
 	interface PHEditingExtensionContext
 	{
 	}
-}
\ No newline at end of file
+
+	interface IPHPickerViewControllerDelegate {}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[Protocol, Model (AutoGeneratedName = true)]
+	[BaseType (typeof(NSObject))]
+	interface PHPickerViewControllerDelegate
+	{
+		[Abstract]
+		[Export ("picker:didFinishPicking:")]
+		void DidFinishPicking (PHPickerViewController picker, PHPickerResult[] results);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (UIViewController))]
+	[DisableDefaultCtor]
+	interface PHPickerViewController
+	{
+		[Export ("configuration", ArgumentSemantic.Copy)]
+		PHPickerConfiguration Configuration { get; }
+
+		[Wrap ("WeakDelegate")]
+		IPHPickerViewControllerDelegate Delegate { get; set; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WeakDelegate { get; set; }
+
+		[Export ("initWithConfiguration:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (PHPickerConfiguration configuration);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	interface PHPickerConfiguration : NSCopying
+	{
+		[Export ("selectionLimit")]
+		nint SelectionLimit { get; set; }
+
+		[NullAllowed, Export ("filter", ArgumentSemantic.Copy)]
+		PHPickerFilter Filter { get; set; }
+
+		[Export ("initWithPhotoLibrary:")]
+		IntPtr Constructor (PHPhotoLibrary photoLibrary);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PHPickerFilter : NSCopying
+	{
+		[Static]
+		[Export ("imagesFilter")]
+		PHPickerFilter ImagesFilter { get; }
+
+		[Static]
+		[Export ("videosFilter")]
+		PHPickerFilter VideosFilter { get; }
+
+		[Static]
+		[Export ("livePhotosFilter")]
+		PHPickerFilter LivePhotosFilter { get; }
+
+		[Static]
+		[Export ("anyFilterMatchingSubfilters:")]
+		PHPickerFilter AnyFilterMatchingSubfilters (PHPickerFilter[] subfilters);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
```

Comment:
`__attribute__((objc_subclassing_restricted))`
Let's `[Seal]` this

Reply:
filed with sharpie (for future bindings) https://github.com/xamarin/ObjectiveSharpie/issues/138

Reply:
```build/ios/native/PhotosUI/PHPickerConfiguration.g.cs(85,22): warning CS0628: 'PHPickerConfiguration.PHPickerConfiguration(IntPtr)': new protected member declared in sealed class
build/ios/native/PhotosUI/PHPickerConfiguration.g.cs(109,27): error CS0549: 'PHPickerConfiguration.Copy(NSZone?)' is a new virtual member in sealed class 'PHPickerConfiguration'
build/ios/native/PhotosUI/PHPickerFilter.g.cs(71,22): warning CS0628: 'PHPickerFilter.PHPickerFilter(IntPtr)': new protected member declared in sealed class
build/ios/native/PhotosUI/PHPickerFilter.g.cs(79,27): error CS0549: 'PHPickerFilter.Copy(NSZone?)' is a new virtual member in sealed class 'PHPickerFilter'
build/ios/native/PhotosUI/PHPickerResult.g.cs(71,22): warning CS0628: 'PHPickerResult.PHPickerResult(IntPtr)': new protected member declared in sealed class
build/ios/native/PhotosUI/PHPickerViewController.g.cs(86,22): warning CS0628: 'PHPickerViewController.PHPickerViewController(IntPtr)': new protected member declared in sealed class
```

Reply:
Replacing with `[Advice ("This type should not be subclassed.")]` for now :)